docs/8.project/project-structure.mdx

```diff
@@ -51,7 +51,7 @@ You can go a long way capturing user inputs using regular expressions, but it ob
 
 NLU lets you capture user inputs by "intent" instead of parsing its raw text. An intent represents all the different ways users can express a unit of meaning that is valid for your bot. For example, you can group the sentences "What's the weather in California?" and "Do you know if it's sunny today in California?" to the intent `GetWeather` and the parameter `city=California`. You can then map that intent to an action using a route.
```

Comment:
maybe in a future PR we should define a style guideline: 
* eg. how to highlight technical words like "Intent" and "Entity"? with double quotes, capitals, italics...?